src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -246,6 +246,8 @@ public function getTestsForParse()
             array('{\'foo\': \'bar\', "bar": \'foo: bar\'}', array('foo' => 'bar', 'bar' => 'foo: bar')),
             array('{\'foo\'\'\': \'bar\', "bar\"": \'foo: bar\'}', array('foo\'' => 'bar', 'bar"' => 'foo: bar')),
             array('{\'foo: \': \'bar\', "bar: ": \'foo: bar\'}', array('foo: ' => 'bar', 'bar: ' => 'foo: bar')),
+            array('{foo:bar: "baz"}', array('foo:bar' => 'baz')),
```

Comment:
as it was accepted before, we should indeed do it in master.

changing the behavior in a 2.7 patch release is too likely to break things silently.